CriticalMass/BikeAnnotationView.swift

```diff
@@ -41,82 +41,54 @@ class BikeAnnoationView: MKAnnotationView {
         backgroundColor = .clear
 
         frame = Constants.shapeRect
-
         layer.addSublayer(ovalShapeLayer)
-    }
-}
-
-class FriendAnnotationView: MKAnnotationView {
-    var friendView: FriendView?
 
-    var friend: Friend? {
-        didSet {
-            friendView?.name = friend?.name ?? ""
+        if #available(iOS 11.0, *) {
+            clusteringIdentifier = BikeClusterAnnotationView.clusteringIdentifier
         }
     }
+}
+
+@available(iOS 11.0, *)
+class BikeClusterAnnotationView: MKAnnotationView {
```

Comment:
the diff is super difficult to read 🙈 You basically added the `BikeClusterAnnotationView` and moved `FriendAnnotationView` and `FriendView` into an own file, right?

Reply:
Yeah. That was probably not the best idea

Reply:
Yeah. That was probably not the best idea

Reply:
moved it the cluster annotation view to its own file 📁